src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Validation/HealthCertificateValidationViewModel.swift

```diff
@@ -14,28 +14,176 @@ final class HealthCertificateValidationViewModel {
 		healthCertificate: HealthCertificate,
 		countries: [Country],
 		store: HealthCertificateStoring,
-		onValidationButtonTap: @escaping (Country, Date) -> Void
+		onValidationButtonTap: @escaping (Country, Date) -> Void,
+		onDisclaimerButtonTap: @escaping () -> Void,
+		onInfoButtonTap: @escaping () -> Void
 	) {
 		self.healthCertificate = healthCertificate
 		self.countries = countries
 		self.store = store
 		self.onValidationButtonTap = onValidationButtonTap
+		self.onDisclaimerButtonTap = onDisclaimerButtonTap
+		self.onInfoButtonTap = onInfoButtonTap
 	}
 
 	// MARK: - Internal
 
+	enum CellIdentifiers: String, TableViewCellReuseIdentifiers {
+		case countrySelectionCell = "CountrySelectionCell"
+		case validationDateSelectionCell = "ValidationDateSelectionCell"
+		case legalDetails = "DynamicLegalCell"
+	}
+
 	var selectedArrivalCountry = Country.defaultCountry()
 	var selectedArrivalDate = Date()
 
 	func validate() {
 		onValidationButtonTap(selectedArrivalCountry, selectedArrivalDate)
 	}
 
+	var dynamicTableViewModel: DynamicTableViewModel {
+		DynamicTableViewModel([
+			.section(
+				cells: [
+					.space(height: 8),
+					.body(
+						text: AppStrings.HealthCertificate.Validation.body1,
+						accessibilityIdentifier: ""
+					),
+					.space(height: 8),
+					.headline(
+						text: AppStrings.HealthCertificate.Validation.headline1,
+						accessibilityIdentifier: ""
+					),
+					countrySelectionCell(),
+					.space(height: 8),
+					validationDateSelectionCell(),
+					.space(height: 8),
+					.body(
+						text: AppStrings.HealthCertificate.Validation.body2,
+						accessibilityIdentifier: ""
+					),
+					.headline(
+						text: AppStrings.HealthCertificate.Validation.headline2,
+						accessibilityIdentifier: ""
+					),
+					.bulletPoint(text: AppStrings.HealthCertificate.Validation.bullet1),
+					.space(height: 8),
```

Comment:
Just mentioning: there is a `spacing: .large` option on the `.bulletPoint` cell, the you could probably remove the space after them

Reply:
Good hint. Changed